Resources/Prototypes/Entities/Structures/Power/smes.yml

```diff
@@ -49,3 +49,11 @@
       energy: 1.6
       color: "#c9c042"
       castShadows: false
+    - type: Construction
+      graph: machine
+      node: machine
+    - type: Wires
+      BoardName: "SMES"
+      LayoutId: SMES
```

Comment:
does this layout ID even do anything, i dont think the SMES wire layout exists? if it doesnt just remove this for now